e2e_test/source/cdc/cdc.share_stream.slt

```diff
@@ -383,6 +414,12 @@ select * from numeric_list_to_rw_int256_list_shared order by id;
 1 {NULL,6,57896044618658097711785492504343953926634992332820282019728792003956564819967,NULL,NULL}
 2 {NULL,NULL,NULL}
 
+query II
+select * from numeric_list_to_numeric_list_shared order by id;
+----
+1 {3.14,6,NULL,NULL,NULL}
+2 {NaN,Infinity,-Infinity}
```

Comment:
Why the output for this case is not `{NAN, POSITIVE_INFINITY, NEGATIVE_INFINITY}`? Which is contradict to the code in `pg_numeric_to_varchar()`

Reply:
After double checking, the output for this row is `NULL` now, since debezium will directly convert a numeric list containing `inf`, `-inf` or `nan` to `NULL`.